dlt/common/storages/load_package.py

```diff
@@ -26,13 +28,65 @@
 from dlt.common.schema.typing import TStoredSchema, TTableSchemaColumns
 from dlt.common.storages import FileStorage
 from dlt.common.storages.exceptions import LoadPackageNotFound
-from dlt.common.typing import DictStrAny, StrAny, SupportsHumanize
+from dlt.common.typing import DictStrAny, SupportsHumanize
 from dlt.common.utils import flatten_list_or_items
+from dlt.common.versioned_state import (
+    generate_state_version_hash,
+    bump_state_version_if_modified,
+    TVersionedState,
+    default_versioned_state,
+)
+from typing_extensions import NotRequired
+
+
+class TLoadPackageState(TVersionedState, total=False):
+    created: int
```

Comment:
should I put a datetime string here or is utc timestamp ok?

Reply:
hmmm we use ISO timestamps everywhere

Reply:
changed